packages/mobile/ios/AudiusReactNative/Info.plist

```diff
@@ -1,104 +1,104 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what's up with the info.plist change?

Reply:
yeah it's just my editor auto-formatting. i hade to update version so we didnt get the "updateRequired" stuff

Reply:
but happy to revert the auto-format, and just modify the one line

Reply:
this is fine w/ me